### PR TITLE
Add CI test for GAIA compiler

### PR DESCRIPTION
**Pull request by shirly121**
## Related issue number

Fixes

### CODE REVIEW

File: .github/workflows/local.yml [hunk 1]
```diff
@@ -126,6 +126,15 @@ jobs:
         rm -rf ~/.cargo
         python3 -m pytest -s -v /tmp/test_run_locally.py
 
+    -name: Run Gaia on Maxgraph Store Test
+     run:
```
Comment (lidongze0629) [inline .github/workflows/local.yml:130]: run: |

File: interactive_engine/src/gaia-adaptor/gremlin_test.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+base_dir=$(cd `dirname $0`; pwd)
+ps -ef | grep "com.alibaba.graphscope.gaia.MaxNode" | grep -v grep | awk '{print $2}' | xargs kill -9
+cd ${base_dir}/../.. && mvn clean install -DskipTests -Pv2
+cd ./distribution/target/ && tar xvzf maxgraph.tar.gz && cd maxgraph
+# start server
+LOG_NAME=maxnode MAXGRAPH_CONF_FILE=conf/sample.config ./bin/max_node_gaia.sh
+sleep 20
```
Comment (lidongze0629) [inline interactive_engine/src/gaia-adaptor/gremlin_test.sh:8]: We shouldn't rely on sleep 20s here, can we remove it ?

File: interactive_engine/src/gaia-adaptor/gremlin_test.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (lidongze0629) [inline interactive_engine/src/gaia-adaptor/gremlin_test.sh:1]: I suggest writing the test in the `local.yml` file instead of hiding it in the script

Comment (shirly121) [reply interactive_engine/src/gaia-adaptor/gremlin_test.sh:1]: 可能不同命令要去不同目录，我是想通过base_dir这个shell变量来进到不同目录

File: interactive_engine/src/gaia-adaptor/gremlin_test.sh (hunk 2 above)
Comment (shirly121) [reply interactive_engine/src/gaia-adaptor/gremlin_test.sh:8]: 这个可以先这样吗？我之后再尝试改掉

**Pull request merged**